core/domain/prod_validation_jobs_one_off.py

```diff
@@ -164,7 +164,7 @@ def _get_model_domain_object_instance(cls, unused_item):
             unused_item: ndb.Model. Entity to validate.
 
         Returns:
-            *: A domain object to validate.
+            (*). A domain object to validate.
```

Comment:
I think we typically just do `*.` (reserving the parens for denoting tuples).

Reply:
Done